### ISSUE TITLE
Version 2.4.1 Installed on D: but using Libraries on C:

### ISSUE THREAD

**Issue by cjmoorejr** [parallaxinc/Propeller-Tool#72]
Even though I installed 2.4.1 on the D: drive this application creates, copies and uses library files on the C: drive.  I even modified the registry entry suggested by JonnyMac and it continued to use the C: drive.

Device name	
Processor	Intel(R) Core(TM) i5-1035G4 CPU @ 1.10GHz   1.50 GHz
Installed RAM	8.00 GB (7.60 GB usable)
Device ID	
Product ID	
System type	64-bit operating system, x64-based processor

Edition	Windows 10 Home
Version	20H2
Installed on	‎9/‎4/‎2020
OS build	19042.746
Serial number	
Experience	Windows Feature Experience Pack 120.2212.551.0

**Comment by PropGit** [comment]
Thank you @cjmoorejr 

Unlike v1.3.x and prior, Propeller Tool v2.4+ stores its application files and library files separately (the latter going to the user's My Documents\Propeller Tool folder).

I'm curious if you were installing to D: because of the advice given to install Propeller Tool in a folder outside of the Program Files folder area to avoid the library file problem noted in versions previous to this?  If so, that problem has been solved by this new behavior and a special install is no longer needed strictly to fix that.  

Regardless, I can see that the behavior you noted is not idea from a user whose desire is to install the software and library (and user workspace) elsewhere (on another drive, for example).

This is something that will need an additional user prompt perhaps and some software settings to store preferences.  I'll create separate issues for parts of this and link them here.

**Comment by cjmoorejr** [comment]
No, that's not my reason(s).  I own a Microsoft Surface Pro.  This is my
second actually.  My two reasons for using D: instead of C: are:

1) My initial Surface had a small [IIRC 64GB] C: drive.  I was constantly
uninstalling software and cleaning the hard drive to make room for new
software.  When I got my new Surface I immediately installed a 64GB microSD
card [as D:] and any software that allowed me, got installed on that card.
If it allowed me to define where my home directory was at [or where its
data should be stored], even better.  The Propeller Tool allows me to
install the software on D: but it does NOT allow me to specify where to put
the data (i.e. /Propeller Tools).  So I thought I would just copy the
folder onto D: and delete it from C:  but as you know every time Propeller
Tool starts up if the data directory is missing from C: it [re]creates it.
Even though I could change the drive letter in the registry for the library
location, Propeller Tool kept showing it at C: and kept creating the data
directory on C: as well.  If Propeller Tool would use the library location
in the registry including the drive letter it seemed this issue would be
resolved.

2) When my original surface crapped out everything on the C: drive prior to
my last backup was locked inside the tablet on the SSD.  To make things
even worse my Western Digital backup tool encrypted the data by some sort
of drive code so I couldn't recover any files to my new Surface.  So I had
to disassemble the old Surface to retrieve the embedded solid state drive
and its data.  I don't know if you have ever had the pleasure of  doing
this but it was a PITA.  So, when I got  my new one I started storing all
my data on the D: drive (i.e. microSD card).
